### DISCUSSION TITLE
InstallAndroidDebug gives errors  (Korge 5.3.2)

### DISCUSSION THREAD

**Discussion by meoowe** [korlibs/korge#2163] [Q&A] [answered]
Following the official documentation, I attempted to generate an APK file using the InstallAndroidDebug Gradle task. However, I encountered 55 DexBuilderDebug errors ("Error while Dexing", "type defined multiple times") preventing APK generation. Other Gradle tasks function normally.

**Comment by soywiz** [comment]
Which korge version are you using? You can find it in `build/libs.versions.toml`

**Reply by meoowe** [reply]
I'm using korge 5.3.2 (Sorry for not replying)

**Reply by soywiz** [reply]
Which operating system?
Do you have Android Studio installed? Which SDK version.
And can you provide the full log?

We can use that information to try to reproduce the problem.

**Reply by meoowe** [reply]
@soywiz I am using Windows 11
Dont have Android Studio installed, I am using IntelliJ idea with the Android and Android Design Tools plugins installed
The code is in [this](https://github.com/meoowe/ClickyCat)
And how can I find the full log?
Here are the screenshots of the error
![image](https://github.com/korlibs/korge/assets/96916064/44031400-ea4e-4ef0-b0e4-ddb24baf1997)
![image](https://github.com/korlibs/korge/assets/96916064/3cff59d5-0a41-4876-be73-1b266922484f)
I get the same set of errors on the hello world starter project

**Reply by meoowe** [reply]
After updating to korGE 5.4.0, instead of getting those errors, I get a single error : Android resource linking failed. In line 16 of AndroidManifest.xml (`<profileable android:shell="true" />`) it can't find the attribute android:shell. I can't remove the attribute because it is in the build folder.

**Reply by meoowe** [reply, marked as answer]
I fixed it. This line on settings.gradle.kts (`androidSdk(compileSdk = 28, minSdk = 22, targetSdk = 32)`) was the problem. I removed it and it all worked again.
